packages/reporter/src/errors/error-stack.tsx

```diff
@@ -42,6 +43,10 @@ const ErrorStack = observer(({ err }: Props) => {
 
   let stopLinking = false
   const lines = _.map(stackLines, (stackLine, index) => {
+    if ((stackLine as ParsedStackFileLine).column) {
```

Comment:
Why only decode the file paths if there is a `column` property?

Reply:
There are 2 types of `stackLine`: `ParsedStackFileLine` and `ParsedStackMessageLine`. And one of the properties that `ParsedStackFileLine` has but `ParsedStackMessageLine` doesn't is `column`. I wanted to say "if the type of stackLine is ParsedStackFileLine". 

It's the opposite of `(stackLine as ParsedStackMessageLine).message != null` below. 

I didn't leave any comments because none is used for `(stackLine as ParsedStackMessageLine).message != null` lines. But it may be better to have one.

Reply:
I see. I refactored it a bit to abstract the `isMessageLine()` check and also made that specific condition unnecessary because once you get farther down in this function, it's guaranteed the line will be a `ParsedStackFileLine` (we return early if it's a message line). 

One thing to keep in mind with a condition like that is that if `column === 0`, the condition will falsely fail, so `column != null` would be a safer way to go.